release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -151,6 +151,11 @@ See xref:../installing/installing_openstack/installing-openstack-installer-sr-io
 Installing a cluster on OpenStack that supports SR-IOV-connected compute machines
 ] for more information.
 
+[id="installation-ironic-agent-vlan-support"]
+==== Ironic Python Agent support for vlan interfaces
```

Comment:
Not sure if these should be "VLAN", since I see both usage elsewhere in docs.

Reply:
I actually prefer caps, so will do that